schedule/yast/remote_ssh/remote_ssh_controller_sle15.yaml

```diff
@@ -20,7 +20,7 @@ schedule:
   - installation/user_settings
   - installation/user_settings_root
   - installation/resolve_dependency_issues
-  - installation/installation_overview
+  - installation/installation_settings/validate_default_target
```

Comment:
no rest-api here and we are not doing anything: https://openqa.suse.de/tests/7923102#step/installation_overview/1

Reply:
Turned back.